include/linux/genetlink.h

```diff
@@ -0,0 +1,89 @@
+#ifndef _UAPI__LINUX_GENERIC_NETLINK_H
```

Comment:
Q: What is the license here? is this something copied from the Linux kernel, or created as a part of GSOC?